src/Couchbase.Lite.Shared/API/Document/Blob.cs

```diff
@@ -291,13 +334,18 @@ internal void FLEncode(FLEncoder* enc)
                 // This blob is attached to a document, so save the full metadata
                 var document = GCHandle.FromIntPtr((IntPtr) extra).Target as MutableDocument;
                 var database = document.Database;
-                try {
-                    Install(database);
-                } catch (Exception) {
-                    WriteLog.To.Database.W(Tag, "Error installing blob to database, throwing...");
-                    throw;
+                if (Digest != null && _db == null) {
+                    _db = database;
```

Comment:
I have a feeling this logic would be better off inside of `Install` but it works here too as long as there are not other places which also need it.

Reply:
Moved.